src/python/gudhi/delaunay_complex.pyx

```diff
@@ -99,63 +102,65 @@ cdef class DelaunayComplex:
             del self.this_ptr
 
     def _is_defined(self):
-        """Returns true if DelaunayComplex pointer is not NULL.
+        """Returns `True` if DelaunayComplex pointer is not NULL.
          """
         return self.this_ptr != NULL
 
-    def create_simplex_tree(self, max_alpha_square = float('inf'), filtration = None):
+    def create_simplex_tree(self, double max_alpha_square: float = float('inf'),
+                            filtration: Optional[Literal['alpha', 'cech']] = None,
+                            bool output_squared_values: bool = True) -> SimplexTree:
         """
-        :param max_alpha_square: The maximum alpha square threshold the simplices shall not exceed. Default is set to
-            infinity, and there is very little point using anything else since it does not save time.
-        :type max_alpha_square: float
-        :param filtration: Set this value to `None` (default value) if filtration values are not needed to be computed
-            (will be set to `NaN`). Set it to `alpha` to compute the filtration values with the Alpha complex, or to
-            `cech` to compute the Delaunay Cech complex.
-        :type filtration: string or None
-        :returns: A simplex tree created from the Delaunay Triangulation. The vertex `k` corresponds to the k-th input
-            point. The vertices may not be numbered contiguously as some points may be discarded in the triangulation
-            (duplicate points, weighted hidden point, ...).
-        :rtype: SimplexTree
+        Args:
+            max_alpha_square: The maximum alpha square threshold the simplices shall not exceed. Default is set to
+                infinity, and there is very little point using anything else since it does not save time.
+            filtration: Set this value to `None` (default value) if filtration values are not needed to be computed
+                (will be set to `NaN`). Set it to `alpha` to compute the filtration values with the Alpha complex, or
+                to `cech` to compute the Delaunay Cech complex.
+            output_squared_values: Square filtration values when `True`. Default is `True`.
```

Comment:
Does the Python doc mention somewhere that passing `False` can be dangerous with negative weights?